### PR TITLE
New homepage what is juju

### PR DESCRIPTION
**Pull request by anthonydillon**
## Done
Build the "What is Juju" animated section of the homepage

## QA
- Open the demo link
- Scroll down the page and through the "What is Juju" section and see the animated illustrations
- Check different screen sizes.

## Issue / Card
Fixes https://github.com/canonical-web-and-design/snap-squad/issues/2017

### CODE REVIEW

**Comment by tbille** [comment]
@anthonydillon if you scroll down and scroll back up super fast the right assets stay with me

superfast

**Comment by anthonydillon** [comment]
@tbille good spot, fixed :+1:

File: static/js/what-is-juju.js [hunk 1]
```diff
@@ -0,0 +1,49 @@
+
```
Comment (tbille) [inline static/js/what-is-juju.js:1]: hello line :wink:

**Review by tbille — APPROVED** [review]
QA+1 on mobile + different browsers.
I looks really good!!

**Comment by danielmutis** [comment]
@anthonydillon when the first illustration locks in place it moves slightly to the right.

**Comment by danielmutis** [comment]
@anthonydillon 1440px and on chrome

**Comment by spencerbygraves** [comment]
Thanks @anthonydillon, a couple of thoughts from me.

- Can we use the [dark horizontal rule](https://vanillaframework.io/docs/examples/base/hr-dark) under each number please
- The transition back and forth between image 4 and 5 seems a little jumpy, the others are good
- Could the last image fade before the outro appears? It overlaps slightly

File: templates/partials/_what-is-juju.html [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+  </div>
+  <div class="row u-full-height u-vertically-center">
+    <div class="col-3 p-heading-highlight">
+      <h3 class="p-heading-highlight__item p-heading--two">5.</h3>
+      <hr class="is-dark">
+      <p>Models, cross-model relations, and model driven operations give you the control to handle deployments and operations at scale, across hybrid clouds, via CLI, or in a visual tool such as the Juju GUI or JAAS.</p>
+      <p><a class="p-button--positive" href="/docs">Check out the docs</a></p>
+    </div>
+    <div class="col-9 u-hide--x-large">
+      <img src="https://assets.ubuntu.com/v1/a87fb8bf-5_full-panel_new.png" alt=""/>
```
Comment (solazio) [inline templates/partials/_what-is-juju.html:124]: can you not use the image template?

**Review by solazio — COMMENTED** [review]
Looks great @anthonydillon ! Just a few comments in the code, and a few more here:
- [x] the js is not transpiled, so you will have to give up on `const`, `let`, etc, or add babel
- [ ] can you not use the image template for the images?

File: static/js/what-is-juju.js [hunk 3]
```diff
@@ -0,0 +1,48 @@
+let lastKnownScrollPosition = 0;
+let ticking = false;
+let stage = 0;
+const animation = document.querySelector('.p-feature');
+const title = document.querySelector('.p-feature__title');
+
+function setStage(scrollPos) {
```
[the remark this answers is not in the document]
Comment (anthonydillon) [reply static/js/what-is-juju.js:7]: Can't without transpiling

File: static/sass/_patterns_heading-highlight.scss [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+      display: block;
+      left: 0;
+      top: 0;
+    }
+
+    p.p-heading-highlight__item:before {
+      top: -1.2rem;
+    }
+  }
+}
```
[the remark this answers is not in the document]
Comment (anthonydillon) [reply static/sass/_patterns_heading-highlight.scss]: Done

File: static/sass/_patterns_scroller.scss [hunk 5]
```diff
@@ -0,0 +1,273 @@
...
+        &:nth-of-type(11) {
+          animation-name: command11;
+        }
+        &:nth-of-type(12) {
+          animation-name: command12;
+        }
+      }
+    }
+  }
+}
```
[the remark this answers is not in the document]
Comment (anthonydillon) [reply static/sass/_patterns_scroller.scss]: Done

File: templates/partials/_what-is-juju.html [hunk 6]
```diff
@@ -0,0 +1,137 @@
...
+  <div class="row u-full-height u-vertically-center">
+    <div class="col-6 p-heading-highlight">
+      <p class="p-heading-highlight__item p-heading--four">With software driving your software, your team can take control of all your applications, infrastructure, and environments &mdash; in less time &mdash; without the headaches of YAML sprawl.</p>
+      <p><a class="p-button--positive u-no-margin--bottom" href="/docs">Check out the docs</a></p>
+    </div>
+    <div class="col-5 col-start-large-8 u-hide--small">
+      <img src="https://assets.ubuntu.com/v1/4ae9123c-Juju-logo.svg" alt="Juju logo"/>
+    </div>
+  </div>
+</section>
```
[the remark this answers is not in the document]
Comment (anthonydillon) [reply templates/partials/_what-is-juju.html]: Done

File: templates/partials/_what-is-juju.html (hunk 2 above)
Comment (anthonydillon) [reply templates/partials/_what-is-juju.html]: I have updated the images that didn't break it

**Comment by anthonydillon** [comment]
@solazio addressed your comments

**Review by solazio — APPROVED** [review]
lgtm

**Pull request merged**